### ISSUE TITLE
2012 R2 servers and WAPs cannot be added to 2016 FBL

### ISSUE THREAD

**Issue by mwodnicki** [MicrosoftDocs/windowsserverdocs#780]
It appears that this has not been explicitly documented in the article and has caused some confusion.  Perhaps the article can be updated to state as such.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ebfd725f-1b77-751c-1948-f8f88d26a9bc
* Version Independent ID: 7d9c78d9-7736-8f0a-9636-269125a10d6f
* Content: [Upgrading to AD FS in Windows Server 2016](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/deployment/upgrading-to-ad-fs-in-windows-server#feedback)
* Content Source: [WindowsServerDocs/identity/ad-fs/deployment/upgrading-to-ad-fs-in-windows-server.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/identity/ad-fs/deployment/upgrading-to-ad-fs-in-windows-server.md)
* Service: **windows-server-threshold**
* GitHub Login: @billmath
* Microsoft Alias: **billmath**

**Comment by MoNadtheGrey** [comment]
yes agree, we are on 2012R2 / ADFS 3.0 and were planning to use upgrade to ADFS 2019 (or 2016) to take advantage of Azure MFA build capabilities (without having to install MFA server) but stumbled upon this requirements which are not clearly specified and  are hidden towards the end of the document amongst last steps!

**Comment by IngridAtMicrosoft** [comment]
#please-close

**Comment by PRMerger8** [comment]
The following users don't have valid GitHub IDs, or are not collaborators on this repo: ingridatmicrosoft 
gh issue comment 773 -b ingridatmicrosoft
